packages/nimble-blazor/NimbleBlazor/NimbleTableColumn.cs

```diff
@@ -42,4 +42,10 @@ public abstract class NimbleTableColumn : ComponentBase
     /// </summary>
     [Parameter]
     public int? SortIndex { get; set; }
+
+    /// <summary>
+    /// Gets or sets any additional attributes to set on the element.
+    /// </summary>
+    [Parameter(CaptureUnmatchedValues = true)]
+    public IDictionary<string, object>? AdditionalAttributes { get; set; }
```

Comment:
I have mixed feeling about this, and the current guidance we're proposing in the CONTRIBUTING docs. These attributes primarily service one concern, which is styling. I'm not sure there will ever be real value in having someone apply a `class` or `id` attribute to a particular table column. Still, I don't suppose there's much harm in this, and we obviously can't prevent this on the nimble components themselves, so I guess this is fine.

Reply:
>  if we get around to doing away with AdditionalAttributes for all components and provide strictly typed properties (for things like class and id), would we provide those APIs on this class?

Yea I think so. In an ideal world Blazor lets us expose the Web Component API in a well-typed way directly and we don't need any wrapper components as all. In that world all the features of the HTMLElement base class (including id, class, etc) would be exposed. So I think it makes sense for those properties to be on this class as well.

> obviously can't prevent this on the nimble components themselves, so I guess this is fine

Nor would I really want to. The big value sell of web components is that they are standard DOM elements and behave well with those apis.